libraries/standard/http/src/http_client.c

```diff
@@ -73,21 +89,21 @@ static HTTPStatus_t addHeader( HTTPRequestHeaders_t * pRequestHeaders,
                                size_t valueLen );
 
 /**
- * @brief Receive HTTP response from the transport recv interface.
+ * @brief Receive HTTP response from the transport receive interface.
  *
  * @param[in] pTransport Transport interface.
- * @param[in] pResponse Response buffer.
+ * @param[in] pBuffer Response buffer.
  * @param[in] bufferLen Length of the response buffer.
  * @param[out] Bytes received from the transport interface.
  *
  * @return Returns #HTTP_SUCCESS if successful. If there was a network error or
  * more bytes than what was specified were read, then #HTTP_NETWORK_ERROR is
  * returned.
  */
-HTTPStatus_t receiveHttpResponse( const HTTPTransportInterface_t * pTransport,
-                                  uint8_t * pBuffer,
-                                  size_t bufferLen,
-                                  size_t * pBytesReceived );
+HTTPStatus_t receiveHttpData( const HTTPTransportInterface_t * pTransport,
```

Comment:
Why change the same to data if it receives response (as per the comment)?

Reply:
To be consistent with the sendHttpData() function.